src/main/java/com/example/gamelist/GameController.java

```diff
@@ -24,13 +24,23 @@ public  List<Game> gamelist(){
     }
 
 
+
     @PostMapping("/gamelist")
-    public ResponseEntity<Response> create(@RequestBody GameForm form){
+    public ResponseEntity<GamelistService.Response> create(@RequestBody GameForm form){
```

Comment:
Game listServiceのResponseというクラスは作ってないはずです。
```suggestion
    public ResponseEntity<Response> create(@RequestBody GameForm form){
```

Reply:
<img width="836" alt="スクリーンショット 2023-12-17 11 46 21" src="https://github.com/otyacazyu/gamelist/assets/143002834/b5d50f5e-3f1d-4dd6-b9f4-92764d689d4f">
[06196e8](https://github.com/otyacazyu/gamelist/pull/7/commits/06196e8bbd3892fdd96b1fbe67b872b6bc8a8528)

GamelistServiceを削除、Responseのみに変更しました。

※コミット名を間違えてしまいまったが、後日変更方法を探して変更します。

Reply:
コミットハッシュがうまくはれていませんね。
改善してほしいです。
<img width="1007" alt="スクリーンショット 2023-12-17 13 40 16" src="https://github.com/otyacazyu/gamelist/assets/62045457/dc6ee4d0-48a2-4e79-a603-51c7ac7e0bd5">